pontoon/sync/formats/silme.py

```diff
@@ -118,6 +132,15 @@ def __init__(self, parser, path, source_resource=None):
     def translations(self):
         return self.entities.values()
 
+    def escape_quotes(self, value):
+        def escape(match):
+            return '\\' + match.group(0)
+
+        return re.sub(r'(?<!\\)[' + QUOTES + r']', escape, value)
```

Comment:
Could you, instead of regex, simply do a replace each quote variant? I'm not sure if that'd be faster, slower, or easier to maintain. This might be fine.